app/js/index.js

```diff
@@ -6,6 +6,12 @@ import Event from './components/Event';
 
 const {Route, RouteHandler} = Router;
 
+//Have to manually remove auto-added fb hash
+//see here: https://github.com/jaredhanson/passport-facebook/issues/12#issuecomment-5913711
+if(window.location.hash && window.location.hash === "#_=_" || window.location.hash === "_=_"){
```

Comment:
Not sure what you mean by that? I tried capturing and redirecting in the server js but that didn't work.